Frontend/CO.CDP.OrganisationApp/Pages/Support/Organisations.cshtml.cs

```diff
@@ -0,0 +1,28 @@
+using CO.CDP.Organisation.WebApiClient;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CO.CDP.OrganisationApp.Pages.Support;
+
+public class OrganisationsModel(
+    IOrganisationClient organisationClient,
+    ISession session) : LoggedInUserAwareModel(session)
+{
+    [BindProperty]
+    public string? Title { get; set; }
+
+    [BindProperty]
+    public string? Type { get; set; }
+
+    [BindProperty]
+    public IList<ApprovableOrganisation> Organisations { get; set; } = [];
```

Comment:
You shouldn't need model binding to happen on Title and Organisations from what I can see of your code - you don't want these to be populated from request data since you're assigning values to them yourself?

You could keep the `[BindProperty(SupportsGet = true)]` on Type if you wanted to tell it to bind from the query string. But you don't need to because you've got string type as an argument on the OnGet method

```suggestion
    public string? Title { get; set; }

    public string? Type { get; set; }

    public IList<ApprovableOrganisation> Organisations { get; set; } = [];
```

Reply:
Thanks for the feedback. I've applied those changes and tested they work.